### PR TITLE
pin dependencies to gems with new ruby 2.x only releases

### PR DESCRIPTION
**Pull request by jsvd**
fixes https://github.com/logstash-plugins/logstash-output-google_bigquery/issues/21 for master

### CODE REVIEW

File: logstash-output-google_bigquery.gemspec [hunk 1]
```diff
@@ -19,9 +19,9 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "output" }
 
   # Gem dependencies
-  s.add_runtime_dependency 'google-api-client', '0.8.7'
+  s.add_runtime_dependency 'google-api-client', '~> 0.8.7' # version 0.9.x works only with ruby 2.x
   s.add_runtime_dependency 'logstash-codec-plain'
-  s.add_runtime_dependency 'mime-types', '2.6.2'
+  s.add_runtime_dependency 'mime-types', '~> 2' # last version compatible with ruby 2.x
```
Comment (ph) [inline logstash-output-google_bigquery.gemspec]: This constrains might be a bit big, ~> 2.6 might be better?

**Comment by jsvd** [comment]
@ph I think  ~> 2.6 won't work, https://github.com/elastic/logstash/blob/v2.4.0/Gemfile.jruby-1.9.lock#L557

**Comment by ph** [comment]
@jsvd ah semver.. it will solve all your problems today. LGTM.

**Comment by jsvd** [comment]
wait, actually both `~> 2.6` and `~> 2` fix the issue, but `~> 2.6.0` won't:

```ruby
jruby-1.7.25 :006 > Gem::Requirement.new("~> 2").satisfied_by?(Gem::Version.new("2.99.1"))
 => true 
jruby-1.7.25 :007 > Gem::Requirement.new("~> 2.6").satisfied_by?(Gem::Version.new("2.99.1"))
 => true 
jruby-1.7.25 :008 > Gem::Requirement.new("~> 2.6.8").satisfied_by?(Gem::Version.new("2.99.1"))
 => false
```

The benefit in `~> 2.6` over `~> 2` is that it prevents installation of 2.[0..6[ which shouldn't happen anyway, so I'll merge as is. thanks for the review

**Pull request closed without merging**